docs/index.md

```diff
@@ -36,6 +33,12 @@
 * [Adding a New Advert Position](./cookbook/adding-a-new-advert-position.md)
 * [Modifying a Template in Administration](./cookbook/modifying-a-template-in-administration.md)
 
+## Kubernetes
+* [Introduction to Kubernetes](/docs/kubernetes/introduction-to-kubernetes.md)
+* [Continuous Integration Using Kubernetes](/docs/kubernetes/continuous-integration-using-kubernetes.md)
+* [Logging on Continuous Integration server running Kubernetes](/docs/kubernetes/logging-on-continuous-integration-server-running-kubernetes.md)
+* [Production on Google Cloud Platform using Kubernetes, Terraform and Kustomize](/docs/kubernetes/kubernetes-production-google-cloud.md)
```

Comment:
The filename is missing "on" :innocent: 
After renaming, the link should be fixed
```suggestion
* [Production on Google Cloud Platform using Kubernetes, Terraform and Kustomize](/docs/kubernetes/production-on-google-cloud-platform-using-kubernetes-terraform-and-kustomize.md)
```

Please review all the links in your new docs - it seems like you forget to change them after the files were renamed